docs/development.rst

```diff
@@ -191,6 +194,13 @@ Internet Explorer 11 support will be dropped
     so we are not supporting it on newer versions,
     starting on 3.0.0.
 
+3.1.0
```

Comment:
This is not the right place to document the version. This document is describing major development releases, not minor releases. We've been updating the changelog page with versions and future versions/RCs.

Reply:
Also, I feel we should bump the version to an RC so users installing this package via Git have the latest RC at hand. We don't need to release to PyPI yet, but that's the next step

Reply:
> Also, I feel we should bump the version to an RC so users installing this package via Git have the latest RC at hand. We don't need to release to PyPI yet, but that's the next step

OK. I pushed 9f6556b with changelog and rc version.

Reply:
This document is a roadmap planning document mostly for development and is meant as an overview. The changelog is definitely where we've been documenting releases, we don't need to update this document with these release notes.

1.1 was only documented because it was significant to the roadmap. We haven't documented any minor versions since.

Reply:
> OK. I pushed 9f6556b with changelog and rc version.

Looks like you might not have run the directions for increasing the version, documented here:

https://sphinx-rtd-theme.readthedocs.io/en/stable/contributing.html#releasing-the-theme

Reply:
You are correct. I followed those steps now and updated this PR with the changes.